Add configurable retries (#1632)
gruntwork-io/terragrunt: docs/_docs/02_features/auto-retry.md

]
```

By default, `auto-retry` tries a maximum of three times to re-run a command, pausing for five seconds between each retry, at which point it will deem the error as not transient, and accept the `terraform` failure.

> However, we can override these defaults too. For example, the following retries up to five times, with 60 seconds in between each retry:

Reviewer: ```suggestion
However, you can override these defaults. For example, the following retries up to five times, with 60 seconds in between each retry:
```
